packages/utils/src/error.ts

```diff
@@ -127,6 +127,9 @@ export function processError(err: any, diffOptions?: DiffOptions) {
   }
   catch {}
 
+  if (typeof err.cause === 'object')
+    processError(err.cause, diffOptions)
```

Comment:
I treated the 2. Regarding the 1, I could have done:

```ts
err.cause = processError(err.cause, diffOptions);
```

But I feel it might be more risky as re-assigning `.cause` is _not_ needed. It may throw. As we alter it in-place during the call, there is no real need to re-assign (re-assigning to itself). I wrote _not_ in italic as actually they are some cases that may justify the attempt to reassign as in some code paths the returned value is not _self_.

Reply:
If I'm ignoring the result I might just split `processError` into two functions: one sanitizing, the other serializing. That's said I'll go for the re-assignment idea for my attempt. Let me know what's the best option in your opinion.